ffi/capi/dart/package/Makefile

```diff
@@ -11,4 +11,3 @@ test/libicu_capi_cdylib.so: ../../../../target/release/libicu_capi_cdylib.so
 	cp ../../../../target/release/libicu_capi_cdylib.so test/libicu_capi_cdylib.so
 
 test: test/libicu_capi_cdylib.so
-	dart test
```

Comment:
don't remove this, it's used in CI. Instead use `make test/libicu_capi_cdylib.so` in the build.dart